### DISCUSSION TITLE
重复MOD过滤

### DISCUSSION THREAD

**Discussion by LemonGasRoot** [Meloong-Git/PCL#4866] [提问] [answered]
![屏幕截图 2024-10-01 194008](https://github.com/user-attachments/assets/551a7490-2d18-4a55-848a-af869720bd86)
自己看吧
一个支持Forge
一个支持Fabric
一个两个都支持
可不可以过滤一下啊

**Comment by Deep-Dark-Forest** [comment, marked as answer]
参考 #1318
